diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index eee227822a772bec128d6adbe4f6aca90ed810f6..eadcc8b75438f37a621a60cf7834f2127af8e6e8 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -83,6 +83,7 @@
   <script type="module" src="chrome://global/content/elements/moz-support-link.mjs"/>
   <script src="chrome://browser/content/migration/migration-wizard.mjs" type="module"></script>
   <script type="module" src="chrome://global/content/elements/moz-toggle.mjs"/>
+  <script src="chrome://browser/content/contentTheme.js"></script>
   <script type="module" src="chrome://global/content/elements/moz-message-bar.mjs" />
   <script type="module" src="chrome://global/content/elements/moz-label.mjs"/>
 </head>
