diff --git a/browser/components/preferences/preferences.xhtml b/browser/components/preferences/preferences.xhtml
index 38fba9a7265cbcec406fd4145014083286fa798e..ec90383adb770f3c023a4f12d1a42d0688f6be6f 100644
--- a/browser/components/preferences/preferences.xhtml
+++ b/browser/components/preferences/preferences.xhtml
@@ -83,6 +83,7 @@
   <script type="module" src="chrome://global/content/elements/moz-support-link.mjs"/>
   <script src="chrome://browser/content/migration/migration-wizard.mjs" type="module"></script>
   <script type="module" src="chrome://global/content/elements/moz-toggle.mjs"/>
+  <script src="chrome://browser/content/contentTheme.js"></script>
   <script type="module" src="chrome://global/content/elements/moz-message-bar.mjs" />
 </head>
 
