diff --git a/browser/components/BrowserGlue.sys.mjs b/browser/components/BrowserGlue.sys.mjs
index 5b47c3952e628c4f4b63bc25ed5563c50a3476dd..24d38d166269ae08bb649451aa0acc853705ca96 100644
--- a/browser/components/BrowserGlue.sys.mjs
+++ b/browser/components/BrowserGlue.sys.mjs
@@ -588,6 +588,8 @@ let JSWINDOWACTORS = {
       "chrome://browser/content/places/historySidebar.xhtml",
       "chrome://browser/content/places/bookmarksSidebar.xhtml",
       "about:firefoxview",
+      "about:*",
+      "chrome://*",
     ],
   },
 
@@ -4468,6 +4470,23 @@ BrowserGlue.prototype = {
   },
 
   async _maybeShowDefaultBrowserPrompt() {
+    // =========================================================================
+    // Pulse browser welcome
+    
+    const welcomeEnabledPref = 'pulse.welcome.enabled'
+    const welcomeSeenPref = 'pulse.welcome.seen'
+    
+    // NOTE: Get bool prefs takes in two options, the pref string and a default
+    // value if it is not defined
+    
+    const welcomeEnabled = Services.prefs.getBoolPref(welcomeEnabledPref, true)
+    const welcomeSeen = Services.prefs.getBoolPref(welcomeSeenPref, false)
+
+    if (welcomeEnabled && !welcomeSeen)
+      this._showWelcomeDialog()
+
+    // =========================================================================
+
     // Highest priority is about:welcome window modal experiment
     // Second highest priority is the upgrade dialog, which can include a "primary
     // browser" request and is limited in various ways, e.g., major upgrades.
@@ -4926,6 +4945,18 @@ BrowserGlue.prototype = {
     "nsIObserver",
     "nsISupportsWeakReference",
   ]),
+
+  /**
+  * Displays the welcome dialog. Triggered `_maybeShowDefaultBrowserPrompt`.
+  *
+  * @todo Add gluon docs for how to show custom popups on startup
+  * @patch Pulse Browser
+  */
+  _showWelcomeDialog() {
+    lazy.BrowserWindowTracker.getTopWindow().gDialogBox.open(
+   "chrome://browser/content/welcome/welcome.html"
+    );
+  },
 };
 
 var ContentBlockingCategoriesPrefs = {
