diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index bfeb59ead8fcd7d17936d4daddf5a3651666a44b..dcd1442b9d39d882f7efda300792a68d2d316f69 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -3030,6 +3030,8 @@ pref("browser.mailto.prompt.os", true);
 // Pref to initialize the BackupService soon after startup.
 pref("browser.backup.enabled", true);
 
+#include pulse-browser.js
+
 // Pref to enable the new profiles
 pref("browser.profiles.enabled", false);
 
