diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 1ef5a05907f4675e4e7191dde12c33dc6da5e3c6..e1e24b276033cc5116e462a8c12ea79c7afa9311 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -3032,6 +3032,8 @@ pref("browser.mailto.prompt.os", true);
 
 pref("browser.backup.enabled", false);
 
+#include pulse-browser.js
+
 // Pref to enable the new profiles
 pref("browser.profiles.enabled", false);
 
