diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 29e5b7316f3549181f4a1415a8324e1bf81519a4..9c909e8e68fc435e4d24e7de4724f476b468b1d9 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -19,6 +19,8 @@
   #endif
 #endif
 
+#include pulse-browser.js
+
 pref("browser.hiddenWindowChromeURL", "chrome://browser/content/hiddenWindowMac.xhtml");
 
 // Set add-ons abuse report related prefs specific to Firefox Desktop.
