diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 8a0558ae2e2d694fdb2a8ed20e712216edd046d1..249c869e70f449bb19710db70a3ef06727826ebe 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -19,6 +19,8 @@
   #endif
 #endif
 
+#include pulse-browser.js
+
 pref("browser.hiddenWindowChromeURL", "chrome://browser/content/hiddenWindowMac.xhtml");
 
 // Enables some extra Extension System Logging (can reduce performance)
