diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index 832388c145cfa6b22cee650f65c82e594f353d82..3903c1bb49e92b19411e48fdec38b164276e7a4a 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -3098,3 +3098,5 @@ pref("toolkit.contentRelevancy.enabled", false);
 pref("toolkit.contentRelevancy.ingestEnabled", false);
 // Pref to enable extra logging for the content relevancy feature
 pref("toolkit.contentRelevancy.log", false);
+
+#include pulse-browser.js
