diff --git a/toolkit/mozapps/extensions/internal/XPIInstall.sys.mjs b/toolkit/mozapps/extensions/internal/XPIInstall.sys.mjs
index 1a80407ad2ad5329b2473dfafcc6012a55a3dd04..5b79ddfbb8ecfbfac8ebcfe1547cbeca8b103b9a 100644
--- a/toolkit/mozapps/extensions/internal/XPIInstall.sys.mjs
+++ b/toolkit/mozapps/extensions/internal/XPIInstall.sys.mjs
@@ -2928,11 +2928,13 @@ export var UpdateChecker = function (
     "resource://gre/modules/addons/AddonUpdateChecker.sys.mjs"
   ));
 
+  if (!aPlatformVersion) aPlatformVersion = Services.appinfo.platformVersion;
+
   this.addon = aAddon;
   aAddon._updateCheck = this;
   XPIInstall.doing(this);
   this.listener = aListener;
-  this.appVersion = aAppVersion;
+  this.appVersion = aPlatformVersion;
   this.platformVersion = aPlatformVersion;
   this.syncCompatibility =
     aReason == AddonManager.UPDATE_WHEN_NEW_APP_INSTALLED;
@@ -2959,7 +2961,7 @@ export var UpdateChecker = function (
     aReason |= UPDATE_TYPE_NEWVERSION;
   }
 
-  let url = escapeAddonURI(aAddon, updateURL, aReason, aAppVersion);
+  let url = escapeAddonURI(aAddon, updateURL, aReason, aPlatformVersion);
   this._parser = AddonUpdateChecker.checkForUpdates(aAddon.id, url, this);
 };
 
