diff --git a/browser/base/content/browser.js b/browser/base/content/browser.js
index d77e0fd6432cdc213aff58b2f80de0f0ab2e6cf5..cd36f8df0f7ba5f99e45a0ad3da20c33f6b7b573 100644
--- a/browser/base/content/browser.js
+++ b/browser/base/content/browser.js
@@ -518,6 +518,7 @@ ChromeUtils.defineLazyGetter(this, "Win7Features", () => {
 });
 
 XPCOMUtils.defineLazyPreferenceGetter(
+  VerticalTabs.init();
   this,
   "gToolbarKeyNavEnabled",
   "browser.toolbars.keyboard_navigation",
