diff --git a/browser/base/content/browser-menubar.inc b/browser/base/content/browser-menubar.inc
index 0c8b4a1cf622247aea8ffd415bcf7be0fe2c9a3d..048b737e4848ab6233ca13dd2375730aa123d140 100644
--- a/browser/base/content/browser-menubar.inc
+++ b/browser/base/content/browser-menubar.inc
@@ -153,6 +153,9 @@
                               type="checkbox"
                               class="sync-ui-item"
                               oncommand="SidebarUI.toggle('viewTabsSidebar');" data-l10n-id="menu-view-synced-tabs-sidebar"/>
+                    <menuitem id="menu_downloadsSidebar"
+                              type="checkbox"
+                              oncommand="SidebarUI.toggle('viewDownloadsSidebar');" data-l10n-id="navbar-downloads"/>
                     <menuitem id="menu_megalistSidebar"
                               type="checkbox"
                               oncommand="SidebarUI.toggle('viewMegalistSidebar');" data-l10n-id="menu-view-megalist-sidebar"/>
