diff --git a/browser/base/content/browser-box.inc.xhtml b/browser/base/content/browser-box.inc.xhtml
index 41258d81bbd2549a2d80cb7df9a9d65aa73a46f1..ffda80fbf62e18b22172c97e486d770077a539e2 100644
--- a/browser/base/content/browser-box.inc.xhtml
+++ b/browser/base/content/browser-box.inc.xhtml
@@ -3,10 +3,14 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 <hbox flex="1" id="browser">
+  <div id="sidebar-container" class="sidebar-container"> 
+    <vbox id="sidebar-icons"></vbox>
+    <vbox id="sidebar-icons-bottom"></vbox>
+  </div>
   <html:sidebar-launcher id="sidebar-launcher" flex="1" hidden="true"></html:sidebar-launcher>
   <vbox id="sidebar-box" hidden="true" class="chromeclass-extrachrome">
     <box id="sidebar-header" align="center">
-      <toolbarbutton id="sidebar-switcher-target" class="tabbable" aria-expanded="false">
+      <toolbarbutton id="sidebar-switcher-target" class="tabbable" aria-expanded="false" flex="1">
         <image id="sidebar-icon" consumeanchor="sidebar-switcher-target"/>
         <label id="sidebar-title" crop="end" control="sidebar"/>
         <image id="sidebar-switcher-arrow"/>
