diff --git a/browser/base/content/browser-menubar.inc b/browser/base/content/browser-menubar.inc
index 0075a610d86749e56f2e344b58d9d44389ba1c6c..d6d34fcbebc90314d2fed5c448157f02ca68dd4f 100644
--- a/browser/base/content/browser-menubar.inc
+++ b/browser/base/content/browser-menubar.inc
@@ -232,6 +232,9 @@
                 <menuitem id="sync-tabs-menuitem"
                           oncommand="gSync.openSyncedTabsPanel();"
                           hidden="true" data-l10n-id="menu-history-synced-tabs"/>
+				<menuitem id="menu_downloadsSidebar"
+                          type="checkbox"
+                          oncommand="SidebarUI.toggle('viewDownloadsSidebar');" data-l10n-id="navbar-downloads"/>
                 <menuitem id="historyRestoreLastSession"
                           command="Browser:RestoreLastSession" data-l10n-id="menu-history-restore-last-session"/>
                 <menuitem id="hiddenTabsMenu"
