diff --git a/browser/app/profile/firefox.js b/browser/app/profile/firefox.js
index e2c00a7d241357ccd227c6a557d8f8c191f0e490..8c710fcf65b82e59cbe1fc7813009e7f7bd62f92 100644
--- a/browser/app/profile/firefox.js
+++ b/browser/app/profile/firefox.js
@@ -13,6 +13,8 @@
 // improves readability, particular for conditional blocks that exceed a single
 // screen.
 
+#include JaduaGo.js
+
 #ifdef XP_UNIX
   #ifndef XP_MACOSX
     #define UNIX_BUT_NOT_MAC
