diff --git a/browser/themes/BuiltInThemeConfig.sys.mjs b/browser/themes/BuiltInThemeConfig.sys.mjs
index 7cfea4b705a1338b1eb5c4f255808aeac4bdb819..c9454a5e5998d05a3ae9cc7d821d35f6233e9e77 100644
--- a/browser/themes/BuiltInThemeConfig.sys.mjs
+++ b/browser/themes/BuiltInThemeConfig.sys.mjs
@@ -20,24 +20,17 @@
  */
 export const BuiltInThemeConfig = new Map([
   [
-    "firefox-compact-light@mozilla.org",
+    "pulse-light@browser.fushra.com",
     {
       version: "1.2",
-      path: "resource://builtin-themes/light/",
+      path: "resource://builtin-themes/pulse/light/",
     },
   ],
   [
-    "firefox-compact-dark@mozilla.org",
+    "pulse-dark@browser.fushra.com",
     {
       version: "1.2",
-      path: "resource://builtin-themes/dark/",
-    },
-  ],
-  [
-    "firefox-alpenglow@mozilla.org",
-    {
-      version: "1.4",
-      path: "resource://builtin-themes/alpenglow/",
+      path: "resource://builtin-themes/pulse/dark/",
     },
   ],
   [
