diff --git a/browser/themes/ThemeVariableMap.sys.mjs b/browser/themes/ThemeVariableMap.sys.mjs
index ff8e85336158e4d86fab047882d6430b9e781445..0c3441c459da4d8f60ceaae75b190219fcb82a4b 100644
--- a/browser/themes/ThemeVariableMap.sys.mjs
+++ b/browser/themes/ThemeVariableMap.sys.mjs
@@ -114,6 +114,7 @@ export const ThemeVariableMap = [
     "--sidebar-background-color",
     {
       lwtProperty: "sidebar",
+	  optionalElementID: "browser",
       processColor(rgbaChannels) {
         if (!rgbaChannels) {
           return null;
@@ -159,6 +160,18 @@ export const ThemeVariableMap = [
       },
     },
   ],
+  [
+    "--in-content-link-color",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
+  [
+    "--in-content-link-color-hover",
+    {
+      lwtProperty: "icon_color",
+    },
+  ],
 ];
 
 export const ThemeContentPropertyList = [
@@ -170,4 +183,10 @@ export const ThemeContentPropertyList = [
   "sidebar_highlight_text",
   "sidebar_text",
   "zap_gradient",
+
+  // Pulse
+  "button_primary",
+  "button_primary_hover",
+  "button_primary_active",
+  "button_primary_color",
 ];
