diff --git a/browser/components/urlbar/UrlbarInput.sys.mjs b/browser/components/urlbar/UrlbarInput.sys.mjs
index dd362cdd8c15ae690f7e54db5c571e262e5eb83e..8459a0ac7eb083597a99f012e26bd09c57f4e40a 100644
--- a/browser/components/urlbar/UrlbarInput.sys.mjs
+++ b/browser/components/urlbar/UrlbarInput.sys.mjs
@@ -455,6 +455,16 @@ export class UrlbarInput {
           (!this.window.isBlankPageURL(uri.spec) ||
             uri.schemeIs("moz-extension") ||
             isInitialPageControlledByWebContent);
+
+	  // Check if the addon is tabliss (our new tab) and stop if from showing up
+      // as such
+      try {
+        const { WebExtensionPolicy } = Cu.getGlobalForObject(Services);
+        const policy = WebExtensionPolicy.getByURI(uri);
+        if (policy.id === 'extension@tabliss.io') {
+          valid = false
+        }
+      } catch (e) {}
       }
     } else if (
       this.window.isInitialPage(value) &&
