diff --git a/toolkit/themes/shared/desktop-jar.inc.mn b/toolkit/themes/shared/desktop-jar.inc.mn
index 34219ec36940f4db25d60da295d3273aac4cbe94..963d8c2aead3cd99caa4deae6980d6a18fb7bc1b 100644
--- a/toolkit/themes/shared/desktop-jar.inc.mn
+++ b/toolkit/themes/shared/desktop-jar.inc.mn
@@ -154,6 +154,9 @@
   skin/classic/global/reader/RM-Content-Width-Plus-44x16.svg           (../../shared/reader/RM-Content-Width-Plus-44x16.svg)
   skin/classic/global/reader/RM-Line-Height-Minus-38x14.svg            (../../shared/reader/RM-Line-Height-Minus-38x14.svg)
   skin/classic/global/reader/RM-Line-Height-Plus-38x24.svg             (../../shared/reader/RM-Line-Height-Plus-38x24.svg)
+  skin/classic/global/icons/bookmark-outline.svg             (../../shared/icons/bookmark-outline.svg)
+  skin/classic/global/icons/tab-rising-edge.svg             (../../shared/icons/tab-rising-edge.svg)
+  skin/classic/global/icons/tab-falling-edge.svg             (../../shared/icons/tab-falling-edge.svg)  
   skin/classic/global/reader/align-left-20.svg             (../../shared/reader/align-left-20.svg)
   skin/classic/global/reader/align-center-20.svg           (../../shared/reader/align-center-20.svg)
   skin/classic/global/reader/align-right-20.svg            (../../shared/reader/align-right-20.svg)
