diff --git a/toolkit/modules/LightweightThemeConsumer.sys.mjs b/toolkit/modules/LightweightThemeConsumer.sys.mjs
index 0d3993370b4ecbdfaf87cfd3145617343c29dc39..58be13f90f7442f066a17272210e7f43c6ee2bb9 100644
--- a/toolkit/modules/LightweightThemeConsumer.sys.mjs
+++ b/toolkit/modules/LightweightThemeConsumer.sys.mjs
@@ -203,6 +203,14 @@ const toolkitVariableMap = [
       },
     },
   ],
+  // Pulse browser: Sidebar colors
+  // Expose some light weight theme variables to be available for use within css
+  [
+    '--lwt-sidebar-background-color',
+    {
+      lwtProperty: "sidebar",
+    }
+  ]
 ];
 
 export function LightweightThemeConsumer(aDocument) {
