diff --git a/browser/themes/shared/jar.inc.mn b/browser/themes/shared/jar.inc.mn
index fb17e3d298472e57e91431ab656b3f688c62053a..8ea95799b2d055eb8f351f2e311e5a6c2ea95f15 100644
--- a/browser/themes/shared/jar.inc.mn
+++ b/browser/themes/shared/jar.inc.mn
@@ -7,6 +7,9 @@
 # be specified once. As a result, the source file paths are relative
 # to the location of the actual manifest.
 
+#include ../pulse/jar.inc.mn
+#include ../designs/jar.inc.mn
+
   skin/classic/browser/aboutFrameCrashed.css                   (../shared/aboutFrameCrashed.css)
   skin/classic/browser/aboutRestartRequired.css                (../shared/aboutRestartRequired.css)
   skin/classic/browser/aboutSessionRestore.css                 (../shared/aboutSessionRestore.css)
