diff --git a/browser/components/extensions/parent/ext-sidebarAction.js b/browser/components/extensions/parent/ext-sidebarAction.js
index 9b5d29929c772ea29a35cf68b035dcc7193b8f86..70ad63e9b433b3958f0c77a9bb4d2dd547eba3e0 100644
--- a/browser/components/extensions/parent/ext-sidebarAction.js
+++ b/browser/components/extensions/parent/ext-sidebarAction.js
@@ -95,6 +95,8 @@ this.sidebarAction = class extends ExtensionAPI {
       if (SidebarController.lastOpenedId === sidebarId) {
         SidebarController.lastOpenedId = null;
       }
+	  // Remove the sidebar item from the sidebar menu.
+      SidebarUI.deleteSidebarItem(sidebarId);
     }
   }
 
@@ -121,11 +123,18 @@ this.sidebarAction = class extends ExtensionAPI {
     if (!this.extension.canAccessWindow(window)) {
       return;
     }
+	
+	let getIcon = size =>
+      IconDetails.escapeUrl(
+        IconDetails.getPreferredIcon(details.icon, this.extension, size).icon
+      );
+	
     this.panel = details.panel;
     let { SidebarController, devicePixelRatio } = window;
     SidebarController.registerExtension(this.id, {
       ...this.getMenuIcon(details, devicePixelRatio),
       menuId: this.menuId,
+	  iconurl: getIcon(16),
       title: details.title,
       extensionId: this.extension.id,
       onload: () =>
@@ -135,6 +144,7 @@ this.sidebarAction = class extends ExtensionAPI {
           this.browserStyle
         ),
     });
+	SidebarUI.createSidebarItem(this.id, SidebarUI.sidebars.get(this.id));
   }
 
   /**
