diff --git a/browser/components/customizableui/CustomizableUI.sys.mjs b/browser/components/customizableui/CustomizableUI.sys.mjs
index f466105b8290b13c672351bb68dd92644dd670fe..1df96069e0807872eb2b4b89eb9e78ac91cd2468 100644
--- a/browser/components/customizableui/CustomizableUI.sys.mjs
+++ b/browser/components/customizableui/CustomizableUI.sys.mjs
@@ -13,6 +13,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
   AddonManagerPrivate: "resource://gre/modules/AddonManager.sys.mjs",
   BrowserUsageTelemetry: "resource:///modules/BrowserUsageTelemetry.sys.mjs",
   CustomizableWidgets: "resource:///modules/CustomizableWidgets.sys.mjs",
+  ExtensionCommon: "resource://gre/modules/ExtensionCommon.sys.mjs",
   HomePage: "resource:///modules/HomePage.sys.mjs",
   PanelMultiView: "resource:///modules/PanelMultiView.sys.mjs",
   PrivateBrowsingUtils: "resource://gre/modules/PrivateBrowsingUtils.sys.mjs",
@@ -26,6 +27,7 @@ ChromeUtils.defineLazyGetter(lazy, "gWidgetsBundle", function () {
 });
 
 const kDefaultThemeID = "default-theme@mozilla.org";
+const kUBlockOriginID = "uBlock0@raymondhill.net";
 
 const kSpecialWidgetPfx = "customizableui-special-";
 
@@ -241,16 +243,18 @@ var CustomizableUIInternal = {
       },
       false
     );
-
+	
+	const { makeWidgetId } = lazy.ExtensionCommon;
     let navbarPlacements = [
       "back-button",
-      "forward-button",
       "stop-reload-button",
+	  "forward-button",
       Services.policies.isAllowed("removeHomeButtonByDefault")
         ? null
         : "home-button",
       Services.prefs.getBoolPref("sidebar.revamp") ? "sidebar-button" : null,
       "spring",
+	  `${makeWidgetId(kUBlockOriginID)}-browser-action`,
       "urlbar-container",
       "spring",
       "save-to-pocket-button",
