diff --git a/browser/locales/en-US/browser/appExtensionFields.ftl b/browser/locales/en-US/browser/appExtensionFields.ftl
index 09d7a076eccd4235fac66156770d8b9ae01a7d16..f6d59214817d1a2dfd290fe06dcf9233583c295c 100644
--- a/browser/locales/en-US/browser/appExtensionFields.ftl
+++ b/browser/locales/en-US/browser/appExtensionFields.ftl
@@ -5,18 +5,56 @@
 ## Theme names and descriptions used in the Themes panel in about:addons
 
 # "Auto" is short for automatic. It can be localized without limitations.
-extension-default-theme-name-auto=System theme — auto
+extension-default-theme-name-auto=Firefox — auto — ⚠️ Broken
 extension-default-theme-description=Follow the operating system setting for buttons, menus, and windows.
 
-extension-firefox-compact-light-name=Light
+extension-firefox-compact-light-name=Firefox (Light)
 extension-firefox-compact-light-description=A theme with a light color scheme.
 
-extension-firefox-compact-dark-name=Dark
+extension-firefox-compact-dark-name=Firefox (Dark)
 extension-firefox-compact-dark-description=A theme with a dark color scheme.
 
 extension-firefox-alpenglow-name=Firefox Alpenglow
 extension-firefox-alpenglow-description=Use a colorful appearance for buttons, menus, and windows.
 
+## Pulse themes
+extension-pulse-light-name=Jadua Classic (Light)
+extension-pulse-light-description=A standard, practical light theme
+
+extension-pulse-dark-name=Jadua Classic (Dark)
+extension-pulse-dark-description=A standard, practical dark theme
+
+# Duplicates here for legacy reasons
+extension-light-name=Jadua Classic (Light)
+extension-light-description=A standard, practical light theme
+
+extension-dark-name=Jadua Classic (Dark)
+extension-dark-description=A standard, practical dark theme
+
+extension-pulse-light-new-name=Jadua (Light)
+extension-pulse-light-new-description=A clean, minimalist light theme
+
+extension-pulse-dark-new-name=Jadua (Dark)
+extension-pulse-dark-new-description=A clean, minimalist light theme
+
+extension-chrome-light-name=Chrome (Light)
+extension-chrome-light-description=A clone of the Google Chrome theme
+
+extension-chrome-dark-name=Chrome (Dark)
+extension-chrome-dark-description=A clone of the Google Chrome theme
+
+extension-gx-light-name=Opera GX (Light)
+extension-gx-light-description=A clone of the Opera GX theme
+
+extension-gx-dark-name=Opera GX (Dark)
+extension-gx-dark-description=A clone of the Opera GX theme
+
+extension-safari-light-name=Safari (Light)
+extension-safari-light-description=A clone of the Safari theme
+
+extension-safari-dark-name=Safari (Dark)
+extension-safari-dark-description=A clone of the Safari theme
+
 ## Colorway Themes
 ## These themes are variants of a colorway. The colorway is specified in the
 ## $colorway-name variable.
